wlct/tournaments.py

```diff
@@ -2422,12 +2422,20 @@ def finish_game(self):
         self.is_finished = True
 
         self.current_state = "Finished"
+        self.tournament = find_tournament_by_id(self.tournament.id)
```

Comment:
Actually on second thought we should never overwrite the tournament member here. I'm going to debug this today and see why it's not able to find it.